pkg/asset/manifests/template.go

```diff
@@ -19,7 +19,8 @@ type AzureCredsSecretData struct {
 
 // OpenStackCredsSecretData holds encoded credentials and is used to generate cloud-creds secret
 type OpenStackCredsSecretData struct {
-	Base64encodeCloudCreds string
+	Base64encodeCloudCreds    string
+	Base64encodeCloudCredsINI string
```

Comment:
nit: I think encode should be capitalized. `Base64encodeCloudCreds` -> `Base64EncodeCloudCreds` for both field names

Reply:
Sure, will be done!

Reply:
Actually I just found that fields of all cloud providers are called in the same manner. Maybe we should just leave them as they are to keep them consistent?https://github.com/openshift/installer/blob/master/pkg/asset/manifests/template.go#L4-L18